spec/ddtrace/contrib/active_record/multi_db_spec.rb

```diff
@@ -133,18 +131,17 @@ def mysql_connection_string
 
           Datadog.configure do |c|
             c.use :active_record, describes: :gadget do |gadget_db|
-              gadget_db.tracer = tracer
               gadget_db.service_name = gadget_db_service_name
             end
 
             c.use :active_record, describes: :widget do |widget_db|
-              widget_db.tracer = tracer
               widget_db.service_name = widget_db_service_name
             end
           end
         end
 
         it do
+          subject
```

Comment:
Minor but you could use the subject name e.g. `count` here instead, and/or move this to the `before` block if all the tests desire this behavior.